internal/client/client.go

```diff
@@ -264,8 +264,14 @@ func (c *CSAPI) SetPushRule(t *testing.T, scope string, kind string, ruleID stri
 // SendEventUnsynced sends `e` into the room.
 // Returns the event ID of the sent event.
 func (c *CSAPI) SendEventUnsynced(t *testing.T, roomID string, e b.Event) string {
-	t.Helper()
 	txnID := int(atomic.AddInt64(&c.txnID, 1))
```

Comment:
```suggestion
	t.Helper()
	txnID := int(atomic.AddInt64(&c.txnID, 1))
```

All helper functions need `t.Helper()` on them.